packages/cloud-cognitive/src/components/Datagrid/styles/addons/_CustomizeColumnsModal.scss

```diff
@@ -13,30 +13,45 @@
     position: inherit;
     display: flex;
     flex-flow: column;
-    /* stylelint-disable-next-line declaration-no-important */
-    padding-top: 0 !important;
+    padding: 0;
+    margin: 0;
   }
 }
 
-.#{$block-class}__customize-columns-checkbox {
+.bx--form-item.bx--checkbox-wrapper.#{$block-class}__customize-columns-checkbox-wrapper {
+  margin-bottom: 0;
+}
+.#{$block-class}__customize-columns-checkbox-wrapper {
   display: flex;
   justify-content: center;
-  /* stylelint-disable-next-line declaration-no-important */
-  margin-bottom: 0 !important; // reset the bottom provided by carbon
 }
 
 .#{$block-class}__customize-columns-modal--actions {
   display: flex;
+  height: $spacing-09;
   flex-flow: row;
-  margin-bottom: $spacing-06;
+  background-color: $field-02;
+}
+
+.#{$block-class}__customize-columns-modal--actions input[role='searchbox'] {
+  height: $spacing-09;
+  padding-left: $spacing-08;
+  border-bottom: none;
+}
+div.#{$block-class}__customize-columns-modal--actions,
+div.bx--search-magnifier,
+.bx--search-magnifier-icon {
```

Comment:
These also need to use the `$carbon-prefix` variable as well

Reply:
added `$carbon-prefix`